lib/Botkit.d.ts

```diff
@@ -217,6 +225,22 @@ declare namespace botkit {
     name: string;
     emails: string[];
   }
+  interface JabberBot extends Bot<JabberSpawnConfiguration, JabberMessage> {
+  }
+  interface JabberConfiguration extends Configuration {
+  }
+  interface JabberController extends Controller<JabberSpawnConfiguration, JabberMessage, JabberBot> {
+  }
+  interface JabberMessage extends Message {
```

Comment:
I see that JabberMessage has custom fields stanza and group, please add them to this interface.
https://github.com/howdyai/botkit/blob/master/lib/JabberBot.js#L192-L195

from_jid field could be useful too.